concepts/architecture/index.md

```diff
@@ -27,7 +27,7 @@ Autonity extends Ethereum at three logical layers:
 	Autonity Protocol smart contracts are part of the client binary. _Liquid Newton_ smart contracts are deployed on validator registration.
 
 - Consensus layer: blockchain consensus provided by the **Proof of Stake Tendermint BFT** protocol. Blocks are proposed by validators and selected by the committee for inclusion in the blockchain, with finality. The consensus mechanism enables dynamic consensus committee selection using a stake-weighting algorithm, maximising the amount of stake securing the system.
-- Communication layer: peer-to-peer networking in the **communication layer** is extended with new block and consensus messaging propagation primitives, to enable the gossiping of information among validators and participant nodes.
+- Communication layer: peer-to-peer networking in the **communication layer** is extended with new block and consensus messaging propagation primitives, to enable the gossiping of transaction and consensus information among validators and participant nodes b [P2P networking protocols](/concepts/system-model/#p2p-networking-protocols).
```

Comment:
typo I guess (the `b` before the link)

Reply:
See commit 81ed1528ebf1e2ce9be557cf9b1ce6ad77a9e4ac